CHANGELOG.md

```diff
@@ -1,23 +1,11 @@
-# NEXT RELEASE
-
-### Enhancements
-* None.
+# 5.19.1 Release notes
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
 * Freelist would keep growing with a decreased commit performance as a result.
   ([2927](https://github.com/realm/realm-sync/issues/2927))
```

Comment:
"Since" forever? @jedelbo

Reply:
"freelist" is an internal implementation detail.
Should it be something like:
"The Realm file could unnecessarily fragmented and cause the file to grow resulting in decreasing commit performance"?